pages/standards-guidelines/index.es.md

```diff
@@ -4,11 +4,11 @@
 title: Resumen de los estándares de accesibilidad de W3C # Do not translate "title:". Do translate the text after "title:".
 nav_title: "Estándares/Pautas" # A short title that is used in the navigation
 lang: es   # Change "en" to the translated language shortcode
-last_updated: 2021-04-30  # Keep the date of the English version
+last_updated: 2024-02-29  # Keep the date of the English version
 
 translation:
   status: outdated
```

Comment:
@remibetin Should I update the status?

Reply:
@cmuncharaz Yes, you can remove the `status: outdated` line.